INSTALL.md

```diff
@@ -411,6 +411,10 @@ To test that the build works, you can try running the test suite, and a one-line
 
 # Disable Lockdown
 
+Note that you may encounter kernel lockdown error if you install bpftrace
+via `snap` incorrectly. Please refer to [Ubuntu](#ubuntu-packages) for more
+details regrading how to use `snap` to install `bpftrace`.
+
```

Comment:
Please move the text to the end of the "Disable Lockdown" section, it will make more sense there.

Reply:
done